sketches/BLE_Scratch/robot.h

```diff
@@ -0,0 +1,26 @@
+#ifndef Robot_h
+#define Robot_h
+
+#include <Arduino.h>
+#include <Servo.h>
+
+class Robot {
```

Comment:
I usually prefer to use `struct` instead, AFAIK the only difference between class and struct is that `struct` has all member public by default, instead the classes have all members private by default.
```suggestion
struct Robot {
```

Reply:
Let's keep `class`